extractor/restriction_parser.cpp

```diff
@@ -152,6 +153,21 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
             {
                 is_only_restriction = true;
             }
+            if (strlen(iter->key()) > 11)

```

Comment:
Why `11` here? Because of the prefix length above? I also would just use string's `.size()` member function.

Hmm are both `key()` and `value()` return values C-strings and not `std::string`? Then I would create a `std::string` for both of them at the beginning an just use these.